ribbon/src/main/java/feign/ribbon/LoadBalancingTarget.java

```diff
@@ -79,7 +94,7 @@ public String name() {
 
   @Override
   public String url() {
-    return name;
+    return scheme + path;
```

Comment:
@adriancole 
Thanks for the comments! I changed this to be `scheme + path` but and all the tests pass but not sure if it's okay to change existing functionality (even if it is "technically" more correct).